packages/components/CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+## 5.0.3 (Unreleased)
```

Comment:
A deprecation must bump the minor version, not patch.

>Minor version Y (x.Y.z | x > 0) should be changed when you add functionality or change functionality in a backwards-compatible manner. It must be incremented if any public API functionality is marked as deprecated.

https://github.com/WordPress/gutenberg/blob/master/CONTRIBUTING.md#maintaining-changelogs